### DISCUSSION TITLE
no connect 50007 port

### DISCUSSION THREAD

**Discussion by czech777** [pcmihnea/HA_WMR500#8] [Q&A]
not connecting to port 50007
device firmware version 1.4.7.6
Wi-Fi connects and telnet does not connect

**Comment by emilic14** [comment]
Hi there, can you make sure that your computer is correctly connected to your device? it should have an IP address of 192.168.10.x
Then also check that your firewall is not blocking the port 50007

The part of the telnet is mostly to configure the device to connect to the wifi with internet (and also get the GUUID and mqtt id, but these you get also later)

**Reply by pcmihnea** [reply]
Also be sure there's no other network connection active on the computer, as it may redirect all traffic request to a internet-capable connection instead. Check with a treaceroute command the wmr500 IP.
I would recommend to do the device configuration after a factory reset, as mentioned in the readme.

**Reply by czech777** [reply]
to pass parameters wifi
first you need to call some command terminal?

**Comment by czech777** [comment]
assigned computer address 192.168.10.2
ping 192.168.10.1 - okey
tried windows 10 debian mac os
disabled firewall

**Reply by czech777** [reply]
traceroute 192.168.10.1
1 ***
2 ***

**Reply by pcmihnea** [reply]
Odd... Btw, How did you find out the firmware version?

**Reply by czech777** [reply]
press select + up

**Comment by czech777** [comment]
changed the firmware to 1.4.9.0
still can't connect to telnet port 50007

**Reply by czech777** [reply]
share your firmware

**Reply by pcmihnea** [reply]
You can find it in the repo.

**Reply by czech777** [reply]
he is modified
i want the original

**Reply by pcmihnea** [reply]
check the latest commit.

**Reply by czech777** [reply]
Thanks

**Reply by emilic14** [reply]
I should be able to share the fw 1.4.7.6 original (if I find in which drive I saved it...)
@pcmihnea what would be the best way for me to share it with you and add it here ?

**Reply by pcmihnea** [reply]
You could do a pull-request, to add it to the `firmware` folder as `wmr500_1476_original.bin`, and mention it in the readme.md?

**Reply by czech777** [reply]
[emilic14](https://github.com/emilic14) Thanks
will wait

**Reply by pcmihnea** [reply]
Branch merged. Thanks!

**Comment by czech777** [comment]
probably the station is not working for me
telnet hanging on connection

**Reply by emilic14** [reply]
I guess you have already tried... but have you factory reset your unit and started from the beginning ?

**Reply by czech777** [reply]
every try
hard reset

**Reply by pcmihnea** [reply]
There might be a possibility the device's WI-FI chip firmware is not up to date. Not sure how to upgrade it
